docker-compose.yml

```diff
@@ -496,6 +496,8 @@ services:
         - OLEFY_THRESHOLD=${OLEFY_THRESHOLD:-5}
         - MAILQ_THRESHOLD=${MAILQ_THRESHOLD:-20}
         - MAILQ_CRIT=${MAILQ_CRIT:-30}
+        - WATCHDOG_SLACK_URL=${WATCHDOG_SLACK_URL}
+        - WATCHDOG_SLACK_PING=${WATCHDOG_SLACK_PING}
```

Comment:
Needs default values (specified with `:-` like on the other variables). Then you don‘t even need to add it to update.sh.

Reply:
Sounds good, I will add the defaults and remove from update.sh

Reply:
Did some further research and I am fairly confident that these values are not candidates for defaults with `:-`. Going to resolve this conversation and ping for review in the main thread. Thanks! 😸

Reply:
They need default values. Otherwise you‘ll get error messages from docker-compose when you bring up Mailcow without these variables in mailcow.conf. So we need a default value for backwards compatibility.

Reply:
@mkuron I added some blank defaults in the latest commit. Let me know what you think.